tests/Fixtures/SucceedingStringLengthValidator.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Tests\Fixtures;
+
+use WMDE\Fundraising\Frontend\Validation\StringLengthValidator;
+use WMDE\Fundraising\Frontend\Validation\ValidationResult;
+
+/**
+ * @license GNU GPL v2+
+ * @author Gabriel Birke < gabriel.birke@wikimedia.de >
+ */
+class SucceedingStringLengthValidator extends StringLengthValidator {
```

Comment:
Yes. I started working on [field length validation](https://phabricator.wikimedia.org/T136604) when I realized that I need to refactor first.